docs/release-notes.md

```diff
@@ -0,0 +1,11 @@
+---
+id: release-notes
+title: Release Notes
+sidebar_label: Release Notes
+---
+
+Release notes are stored on Github.
```

Comment:
nit: maybe make this active, i.e. "See complete release notes on GitHub."

Reply:
Reworded.